ui/src/components/RoleScopesTable/index.jsx

```diff
@@ -139,7 +139,9 @@ export default class RoleScopesTable extends Component {
       </List>
     ) : (
       <Typography variant="body2" className={classes.noRolesText}>
-        No roles available
+        {searchTerm
+          ? `No roles available with search term ${searchTerm}.`
+          : 'No roles available'}
```

Comment:
Do you know why it's not working? Btw, you can access this view in http://localhost:5080/auth/scopes. There are two tables there.